src/app/pages/fund/util.ts

```diff
@@ -0,0 +1,18 @@
+import { useStxBalance } from '@app/common/hooks/balance/stx/use-stx-balance';
+import { useNativeSegwitBalance } from '@app/query/bitcoin/balance/btc-native-segwit-balance.hooks';
+import { createStacksCryptoCurrencyAssetTypeWrapper } from '@app/query/stacks/balance/stacks-ft-balances.utils';
+import { useCurrentAccountNativeSegwitSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+export function useBtcCryptoCurrencyAssetBalance() {
```

Comment:
Not sure a hook should be in a `util` file? We have a pattern for handling hooks using `.hooks.ts`. Also, aren't these exposed for you in a shareable spot already? If not, I would refactor an existing hook, like `useStxBalance` to make available what you need but allow others to use it too? `useAllTransferableCryptoAssetBalances`?

Reply:
Good point! Moved those to the hooks folder